prow/line/line.go

```diff
@@ -334,7 +334,7 @@ func deleteKubeJob(k deleteClient, job kube.Job) error {
 	return nil
 }
 
-func SetJobStatus(k *kube.Client, podName, jobName, state, desc, url string) error {
+func SetJobStatus(k *kube.Client, podName, jobName, state, desc, url, agent string) error {
```

Comment:
We can set agent when the job is created, so that it doesn't need to be reset to the same value on every status change.